core/server/api/webhooks.js

```diff
@@ -0,0 +1,155 @@
+// # Webhooks API
+// RESTful API for creating webhooks
+// also known as "REST Hooks", see http://resthooks.org
+var Promise = require('bluebird'),
+    _ = require('lodash'),
+    https = require('https'),
```

Comment:
Maybe we should add a note to the release, that only https webhook url's are supported atm.

Reply:
Worth putting in the release notes or just in the PR/commit description?

Reply:
I've updated the PR description. This isn't mean as a developer-only feature at the moment so I think the PR description will be fine seeing as it's linked from the release notes.

Reply:
Hmmmmm not sure how important it is. Would be actually cool to mark this feature as lab feature for now?

Reply:
There's no UI for it so I'm not sure where it would go in labs? We don't want a labs toggle for webhook support at the moment as it's needed for Zapier integration, we can certainly mark Zapier as "beta" when the Zapier app is public and we can show templates under a Zapier app in Ghost.

Reply:
> This is meant as a developer-only feature at the moment

How is that communicated? Should we update the commit message?

Reply:
I can update the commit message if needed, the only ways it can be used is

a) adding a webhook record to the DB manually
b) using the API with password auth and POSTing to /webhooks/ (this is private API so not documented)

Reply:
> I can update the commit message if needed

I think that would be useful 👍